Gems/Atom/Bootstrap/Code/Source/BootstrapSystemComponent.cpp

```diff
@@ -176,7 +176,10 @@ namespace AZ
 
                 m_isAssetCatalogLoaded = true;
 
-                RPI::RPISystemInterface::Get()->InitializeSystemAssets();
+                if (!RPI::RPISystemInterface::Get()->IsInitialized())
+                {
+                    RPI::RPISystemInterface::Get()->InitializeSystemAssets();
+                }
 
                 if (!RPI::RPISystemInterface::Get()->IsInitialized())
```

Comment:
Seems odd to have the same if check 5 lines below, can they be combined?

Reply:
I would say no. I only added the if check because two systems are calling this function and it fails if it's already initialized. The second check is checking to make sure initialization succeeded.